README.md

```diff
@@ -184,6 +185,12 @@ cd ~/govuk/content-publisher
 govuk-docker startup e2e
 ```
 
+### Interoperability of stacks
+
+Even if an e2e stack is started, functionality won't necessarily work as expected "end to end" as govuk-docker doesn't mimick the routing we have on GOV.UK. For example, publishing apps that link to draft-origin.dev.gov.uk frontends will see a server error, as draft-origin isn't a service in govuk-docker.
+
+In these cases, you can swap out the URL for the relevant frontend (such as draft-collections.dev.gov.uk). It's also worth noting that this frontend will need to be started in a separate process, as publishing apps don't define frontend apps in their Docker service dependencies; a publishing app doesn't need its corresponding frontend in order to be able to publish.
```

Comment:
This looks good. just a note that 'process' might be a bit confusing, as we're really talking about a 'docker-compose service + dependencies'. Perhaps just say 'started separately'?